documentation/design-docs/dotnet-monitor.yaml

```diff
@@ -0,0 +1,457 @@
+swagger: "2.0"
+info:
+  description: "dotnet-monitor diagnostics agent"
+  version: "1.0.0"
+  title: "dotnet-monitor"
+host: "localhost:9999"
+basePath: "/diag/v1/"
+schemes:
+- "https"
+- "http"
+paths:
+  /processes/:
+    get:
+      tags:
+      - "processes"
+      operationId: "getprocesses"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "Success"
+          schema:
+            type: array
+            items: 
+              $ref: "#/definitions/Process"
+        400:
+          description: "Bad request"
+          schema:
+            $ref: "#/definitions/Error"
+  /dump/:
+    get:
+      tags:
+      - "dumps"
+      operationId: "getdefaultprocessdump"
+      produces:
+      - "application/octet-stream"
+      parameters:
+      - name: "type"
+        in: query
+        description: "Specifies the dump type, which determines the kinds of information that are collected from the process."
+        type: string
+        enum:
+        - "Normal"
+        - "WithHeap"
+        - "Triage"
+        - "Full"
+        required: false
+        default: "Normal"
+      responses:
+        200:
+          description: "Success"
+          schema:
+            format: "byte"
+        400:
+          description: "Bad request"
+          schema:
+            $ref: "#/definitions/Error"
+  /dump/{uid}:
+    get:
+      tags:
+      - "dumps"
+      operationId: "getprocessdump"
+      produces:
+      - "application/octet-stream"
+      parameters:
+      - name: "uid"
+        in: "path"
+        description: "Process ID of the process to collect a process dump"
+        required: true
+        type: "integer"
```

Comment:
May need to change to non-integer type

Reply:
It's a mistake. All other references of uid have already been changed to string